lca-chimevc-stack/cloudformation-templates/chime-vc-call-analytics.yaml

```diff
@@ -67,7 +67,7 @@ Parameters:
     Default: 'false'
     Description: >-
       Enable content redaction from Amazon Transcribe transcription output. This is only used when
-      the 'en-US' language is selected in the TranscribeLanguageCode parameter.
+      the 'en-US', 'en-AU', 'en-GB' or 'es-US' languages are selected in the TranscribeLanguageCode parameter.
```

Comment:
Grammatically, I think this should be: 
`either the 'en-US', 'en-AU', 'en-GB' or 'es-US' language is selected in the TranscribeLanguageCode parameter.`
(Since we support selection of only one language for TranscribeLanguageCode)
But it doesn't matter.. :)